index.js

```diff
@@ -0,0 +1,33 @@
+const item = {
+    itemId: '55sdife7-w32e-5135-abs9-b5632a4ba5b5',
+    name: 'Sweatpants',
+    description: 'Drake Bulldogs 2020 Sweatpants',
+    price: 24.99,
+    size: 'XS, S, M, L, XL, XXL'
```

Comment:
we can make this an array ?
An item can come in different sizes 

```suggestion
    sizes: ['XS, S, M, L, XL, XXL']
```